pkg/netutil/vpn.go

```diff
@@ -0,0 +1,130 @@
+package netutil
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type TailscaleOutput struct {
+	TailscaleIPs []string `json:"TailscaleIPs"`
+}
+
+// VPNInfo includes node information of the VPN. It is a general struct in case we want to add more vpn integrations
+type VPNInfo struct {
+	IPs       []net.IP
+	NodeID    string
+	VPNAPIKey string
+}
+
+// vpnCliAuthInfo includes auth information of the VPN. It is a general struct in case we want to add more vpn integrations
+type vpnCliAuthInfo struct {
+	Name    string
+	JoinKey string
+	APIKey  string
+}
+
+// StartVPN starts the VPN interface. General function in case we want to add more vpn integrations
+func StartVPN(vpnAuthConfigFile string) error {
+	authInfo, err := getVPNAuthInfo(vpnAuthConfigFile)
+	if err != nil {
+		return err
+	}
+
+	if authInfo.Name == "tailscale" {
```

Comment:
This should probably be a switch/default even though we only support one VPN.

Reply:
make sense